41-vaccine-temp-monitor/README.md

```diff
@@ -0,0 +1,321 @@
+                  GND|----------'-^-^----------|GND
+                     |            | |          |
+                  SDA|------------'-^----------|SDA
+                     |              |          |
+                  SCL|--------------'----------|SCL
+                     |                         |
+                   EN|-------------------------|ATTN
+                     |                         |
+      ```
+
+   _**NOTE:** "`'`" indicates a connection and "`^`" indicates a bypass._
+
+   _**NOTE:** You may wish to implement the circuit on a breadboard, to ensure
+   correctness, before soldering._
+1. Plug the Qwiic connector into the Notecarrier-B to make the connections
+described above (leaving only the connection from `OS` to `AUX1` remaining).
+1. Mount the Notecard into the M.2 (key E) connector on the Notecarrier-B.
+1. Connect the antennas to the Notecard as shown in the table below:
+
+   | Notecard U.FL | Antenna                       |
+   |:-------------:|:----------------------------- |
+   |     MAIN      | Molex Antenna 213353 (middle) |
+   |      DIV      | Molex Antenna 209142          |
+   |      GPS      | Molex Antenna 213353 (end)    |
+
+1. Plug the LiPo battery's JST cable into the Notecarrier-B port labeled `LIPO`.
+
+Notecard Firmware Setup
+-----------------------
+
+The Notecard should use
+[firmware version 4.4.1](https://dev.blues.io/notecard/notecard-firmware-updates/#v3-5-1-october-7th-2022)
+or higher. The simplest way to update firmware is to do an
+[over-the-air (OTA) update](https://dev.blues.io/notecard/notecard-firmware-updates/#ota-dfu-with-notehub).
+
+Host Firmware Setup
+-------------------
+
+1. Open Visual Studio Code.
+1. Click the PlatformIO icon on the left hand side and open this project's `firmware/arduino` folder with Quick Access > PIO Home > Open > Open Project.
+1. Open the file `src/main.cpp`.
+1. Uncomment the following line:
+
+   ```c
+   #define PRODUCT_UID "com.your-company:your-product-name"
+   ```
+
+1. Replace `com.your-company:your-product-name` with your ProductUID generated
+   during [Notehub Setup](https://dev.blues.io/notehub/notehub-walkthrough/).
+1. Click the PlatformIO icon on the left hand side again and click "Build" under
+**Project Tasks > featheresp32 > General**.
+
+   You should see "SUCCESS" in the terminal output pane. The firmware image is
+   now ready to be flashed to the host.
+
+1. Click "Upload" under **Project Tasks > featheresp32 > General**.
+
+   You should see "SUCCESS" in the terminal output pane.
+
+Environment Variables
+---------------------
+
+- `alarm_temperature` - The temperature at which the alarm should trigger.
+- `alarm_hysteresis` - The temperature at which the alarm should reset.
+- `sample_interval_s` - The temperature and battery sampling interval in seconds.
+
+_**NOTE:** If you modify any environment variables, you must wait until the next
+synchronization event (max wait: inbound frequency) for the changes to propagate
```

Comment:
I don't think newbs are going to understand what you mean by

> max wait: inbound frequency

Worth explaining.

EDIT: I see you go into this a bit more below, so you could also just mention here that you'll elaborate later in the document.